### PR TITLE
@accounts/boost and docs 

### PR DESCRIPTION
**Pull request by TimMikeladze**
Inspired by `apollo-boost` and `apollo-server`, this PR introduces an `@accounts/boost` package which abstracts the code for setting up an accounts graphql server. 

The idea is to provide some production ready defaults and common authentication services such as passwords with two factor enabled and popular oauth services (facebook, google, twitter). 

Server side consumers of `@accounts/boost` can use `makeRemoteExecutableSchema` if they want to stitch the accounts schema into their app's schema. 

**Changes:**

- Moved contents of `examples` and `docs` repositories into this one. Will archive those repos after this PR is merged. (The old examples and documentation needs to be updated)

- Added getting started with `@accounts/boost` docs to readme  

- Added `@accounts/boost` examples of monolithic and microservice usage

### CODE REVIEW

**Comment by TimMikeladze** [comment]
Updated with getting started documentation.

**Comment by TimMikeladze** [comment]
This PR is ready for review.

**Comment by pradel** [comment]
Amazing @TimMikeladze I will take a look tomorrow 😄

File: packages/graphql-api/src/graphql/mutations.ts [hunk 1]
```diff
@@ -9,8 +9,8 @@ export const mutations = `
 `;
 
 export const mutationsPassword = `
-  # register returns the id corresponding db ids, such as number IDs, ObjectIDs or UUIDs
-  register(user: CreateUserInput!): ID
+  # Creates a user with a password, returns the id corresponding db ids, such as number IDs, ObjectIDs or UUIDs
+  createUser(user: CreateUserInput!): ID
```
Comment (pradel) [inline packages/graphql-api/src/graphql/mutations.ts]: **Breaking change**, need to add in the changelog of the next release

File: examples/accounts-boost/src/microservice/accounts-microservice.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import accountsBoost from '@accounts/boost';
+
+(async () => {
+  const accounts = await accountsBoost({
+    tokenSecret: 'terrible secret',
+  } as any);
```
Comment (pradel) [inline examples/accounts-boost/src/microservice/accounts-microservice.ts]: Can we fix this type?

File: packages/boost/package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "compile": "tsc",
+    "prepublishOnly": "yarn compile"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/accounts-js/accounts/tree/master/packages/boost"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "@accounts/database-manager": "^0.3.0-beta.28",
```
Comment (pradel) [inline packages/boost/package.json]: It's beta 29 now

File: packages/boost/package.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@accounts/boost",
+  "version": "0.3.0-beta.24",
```
Comment (pradel) [inline packages/boost/package.json]: beta 29 :)

File: packages/react-material-ui/src/index.tsx [hunk 5]
```diff
@@ -0,0 +1,191 @@
+import React from 'react';
```
Comment (pradel) [inline packages/react-material-ui/src/index.tsx]: @TimMikeladze can we move these files in another pull request? :)
